package.json

```diff
@@ -39,7 +39,7 @@
     "core-js": "^3.6.1",
     "crypto-browserify": "^3.12.0",
     "css-loader": "^6.4.0",
-    "css-minimizer-webpack-plugin": "3.1.1",
+    "css-minimizer-webpack-plugin": "^4.2.2",
```

Comment:
Hello @zburke 
`"node": ">=14.0.0"` from our app (ui-users)
`"node": ">= 14.15.0"`  from css-minimizer-webpack-plugin (https://github.com/webpack-contrib/css-minimizer-webpack-plugin/blob/master/package.json#L17 , 
https://github.com/webpack-contrib/css-minimizer-webpack-plugin/blob/master/CHANGELOG.md#400-2022-05-18 )

Could you please clarify what we should to do regarding mismatch node version? Can this to be a problem for us?

Reply:
Thank you